### PR TITLE
Added a go ingestor

### PR DESCRIPTION
**Pull request by jsonperl**
Runs 4 times a day at 15 past the hour.

### CODE REVIEW

File: ingestors/golang.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+	"net/url"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/buger/jsonparser"
+	"github.com/librariesio/depper/data"
+)
+
+const golangSchedule = "15 */6 * * *"
```
Comment (katzj) [inline ingestors/golang.go:16]: This seems really infrequent... is it expensive to look more often?

File: ingestors/golang.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+func (ingestor *Golang) Schedule() string {
+	return golangSchedule
+}
+
+func (ingestor *Golang) Ingest() []data.PackageVersion {
+	// Currently the index only shows the last <=2000 package version releases from the date given. (https://proxy.golang.org/)
+	oneDayAgo := url.QueryEscape(time.Now().AddDate(0, 0, -1).Format(time.RFC3339))
+	url := fmt.Sprintf("%s?since=%s&limit=2000", golangIndexUrl, oneDayAgo)
+
+	fmt.Println(url)
```
Comment (tiegz) [inline ingestors/golang.go:36]:
```suggestion
```

File: ingestors/golang.go (hunk 1 above)
Comment (tiegz) [reply ingestors/golang.go:16]: Does seem like waiting 6 hours to see a Module to show up in Libraries might be too long -- would 5 min be too often? (or maybe we could revisit all the ingestors later and tune them after running for a while)

File: ingestors/golang.go (hunk 2 above)
Comment (jsonperl) [reply ingestors/golang.go:36]: Thanks forgot to commit!
Screen Shot 2021-01-22 at 9 51 04 AM

File: ingestors/golang.go (hunk 1 above)
Comment (jsonperl) [reply ingestors/golang.go:16]: We had been doing it once daily :D 5 minutes sounds just dandy! And yes we should tune these at some point.

Comment (jsonperl) [reply ingestors/golang.go:16]: Again sort of staving off making a smarter scheduler.

**Review by tiegz — APPROVED** [review]
:gopher_dance:

**Pull request merged**